movement/movement.c

```diff
@@ -76,6 +82,20 @@
 movement_state_t movement_state;
 void * watch_face_contexts[MOVEMENT_NUM_FACES];
 watch_date_time scheduled_tasks[MOVEMENT_NUM_FACES];
+
+// HPT stuff
+
+// bit flags indicating which watch face wants the HPT enabled.
+// TODO make sure this number width is larger than the number of faces
+#define HPT_REQUESTS_T uint16_t
+HPT_REQUESTS_T hpt_enable_requests;
+
+// timestamps at which watch faces should have a HPT event triggered. UINT64_MAX for no callback
+uint64_t hpt_scheduled_events[MOVEMENT_NUM_FACES];
```

Comment:
Perhaps it would be possible to store this in a watch face structure, eliminating a global variable.

Reply:
I had considered that, but either way, it's going to add up to one 64-bit timestamp per watch face. I didn't see an obvious watch face struct to add it to in movement.c, but I can't say I looked very hard.